test/functionalTest/cases/3000_allow_creation_transient_entities/create_transient_entity.test

```diff
@@ -0,0 +1,115 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
It would be great to include a .test checking error, i.e. trying to create an entity with invalidad dateExpires and check the requests fail with 400 Bad Request.

Reply:
The .test covering error could also include the case of create a dateExpires atribute with a type different from DateTime (e.g. a number, a string, etc.) which I understand also result in 400 Bad Request.

Reply:
Fixed in 8ef8517

Reply:
Second test fixed in fe0bab4